tests/scripts/final_rc_checks/final_rc_checks.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import json
+import argparse
+
+# Set parser variable to reduce redundancy
+parser = argparse.ArgumentParser()
+
+# -fqdn FQDN -u Username -p Password -api API_Bearer_token
+parser.add_argument("-fqdn", "--rancherURL", dest = "fqdn", help="Rancher FQDN e.g. https://ranchertest.qa.rancher.space")
+parser.add_argument("-api", "--api", dest = "api_bearer_token", help="API Bearer token")
+
+args = parser.parse_args()
+
+print( "fqdn {} api {} ".format(
```

Comment:
I would say this part isn't necessary, since you're just printing out the FQDN and api the user just entered.